deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml

```diff
@@ -180,6 +180,17 @@ spec:
                           values:
                             type: array
                             description: set of values for the expression
+            certValidForDays:
+              type: integer
+              description: Number of days generated APIService certs are valid for
+              format: int32
+              minimum: 1
+              maximum: 730
+            certMinFreshSeconds:
+              type: integer
+              description: Minimum freshness of a cert before it is renewed; in seconds
+              format: int32
+              minimum: 10
```

Comment:
(commenting here to so that my comments read correctly in order)

Do we even want to expose these flags? Can't we just pick good defaults and ensure users don't have to worry about them?